test/src/test/java/hudson/console/HyperlinkNoteTest.java

```diff
@@ -35,9 +39,13 @@
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.JenkinsRule;
 
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
+import com.gargoylesoftware.htmlunit.html.HtmlAnchor;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+
 import static org.hamcrest.Matchers.allOf;
 import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertThat;
```

Comment:
Please don't make this change.  The `org.junit.Assert.assertThat` is deprecated in JUnit 4.13 and was recently removed from all uses in Jenkins core.  This is adding a deprecated method that was already configured to use the preferred new method.
```suggestion
import static org.hamcrest.MatcherAssert.assertThat;
```